pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -821,6 +821,8 @@ type PodGC struct {
 	Strategy PodGCStrategy `json:"strategy,omitempty" protobuf:"bytes,1,opt,name=strategy,casttype=PodGCStrategy"`
 	// LabelSelector is the label selector to check if the pods match the labels before being added to the pod GC queue.
 	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty" protobuf:"bytes,2,opt,name=labelSelector"`
+	// DeleteDelaySeconds is how long to delay pod deletion for.
+	DeleteDelaySeconds *int32 `json:"deleteDelaySeconds,omitempty" protobuf:"bytes,3,opt,name=deleteDelaySeconds"`
```

Comment:
I think this should be `uint32` or `metav1.Duration` not pointer type, zero would be delete ASAPA

Reply:
That makes sense will update.